sdk/identity/Azure.Identity/samples/ClientSideUserAuthentication.md

```diff
@@ -76,14 +77,15 @@ Quite often applications desire the ability to be run multiple times without hav
 
 ### Persisting the TokenCache
 
-The `TokenCache` contains all the data needed to silently authenticate, one or many accounts. It contains sensitive data such as refresh tokens, and access tokens and must be protected to prevent compromising the accounts it houses tokens for. The `Azure.Identity` library provides the `PersistentTokenCache` class which by default will protect and persist the cache using available platform data protection.
+The `TokenCache` contains all the data needed to silently authenticate, one or many accounts. It contains sensitive data such as refresh tokens, and access tokens and must be protected to prevent compromising the accounts it houses tokens for. The `Azure.Identity` library provides the `TokenCache` class which by default will protect and persist the cache using available platform data protection.
```

Comment:
Since it's no longer a class we should probably reword the samples not to refer to `TokenCache` directly but talk about credentials caching tokens instead.

Reply:
I took a shot at rewording everything mentioning the `TokenCache`. Let me know how it looks.